documentation/1.3/tour/modules.md

```diff
@@ -18,23 +18,23 @@ We're going to learn about *packages* and *modules*.
 ## Packages and imports
 
 Every source file&mdash;along with everything declared in the 
-file&mdash;belongs to a package. Even source files that don't belong to a 
-named package are considered to belong to the "default" package. It's the 
-location of a source file within a source directory that determines which 
-package a source file belongs to.
```

Comment:
> Even source files that don't belong to a 
 -named package are considered to belong to the "default" package.

Why would you remove this information?